build/patches/Disable-promos-displayed-in-bookmarks-manager.patch

```diff
@@ -5,81 +5,22 @@ Subject: Disable promos displayed in bookmarks manager
 Remove personalized signin promo view from
 bookmarks, never reach signin thresholds.
 ---
- chrome/android/chrome_java_resources.gni      |   1 -
- ...rsonalized_signin_promo_view_bookmarks.xml |  30 -----
- .../bookmarks/BookmarkItemsAdapter.java       |  30 +----
- .../bookmarks/BookmarkPromoHeader.java        | 121 +-----------------
- .../browser/signin/SigninPromoController.java |  23 +---
- 5 files changed, 5 insertions(+), 200 deletions(-)
- delete mode 100644 chrome/android/java/res/layout/personalized_signin_promo_view_bookmarks.xml
+ .../bookmarks/BookmarkItemsAdapter.java       |  29 +----
```

Comment:
For example for this patch there is a moved file in `chrome/browser/signin/ui/android/java/src/org/chromium/chrome/browser/signin/ui/SigninPromoController.java` that was missed; without documenting this, nobody can review it.

Reply:
but you have to see here

https://github.com/uazo/bromite/blob/v89/build/patches/Disable-promos-displayed-in-bookmarks-manager.patch

maybe doing a different commit each time is not good

Reply:
> but you have to see here

Can you explain what to see there? The changes to `SigninPromoController` are missing.

Reply:
I didn't understand.. damn you're right.

i think the problem is in the wiggle options i used, i was missing "--no-ignore", but i realized it too late, when i found myself in a git am in progress, four patches ahead...
maybe I also need something that verifies the number of files modified before and after

I thought I reprocessed it, but once again I have to learn, luckily you checked, nothing escapes you :)

Reply:
At the beginning I was using some of your patches, then I noticed something like this ^ so I stopped; that's why I mentioned the importance of documenting the changes :)

So in this case I thought it was intentional, but without documenting what has been (willingly) changed it is not possible to do this type of review.

Reply:
you're absolutely right, now I understand better what you meant.
I hope they can help at least to see quickly what needs to be changed, because for some I had to go rummaging through the chromium history